src/Security/Authentication/LoginManager.php

```diff
@@ -97,6 +104,8 @@ private function createToken(string $firewallName, UserInterface $user): TokenIn
 
     private function getUserChecker(FirewallConfig $config): UserCheckerInterface
     {
-        return UserCheckerClassResolver::resolveUserChecker($config->getUserChecker());
+        $userChecker = UserCheckerClassResolver::resolveUserChecker($config->getUserChecker());
```

Comment:
still doesn't work because the `$config->getUserChecker()` is passed as `security.user_checker` from symfony security bundle and won't return null

Reply:
even if i use an alias when defining a firewall-level user_checker it doesn't work
https://symfony.com/doc/current/security/user_checkers.html#enabling-the-custom-user-checker